CoreScriptsRoot/Modules/Settings/Pages/Players.lua

```diff
@@ -440,13 +440,14 @@ local function Initialize()
 		nameLabel.ZIndex = 3
 		nameLabel.Parent = frame
 
-		frame.MouseEnter:connect(function()
-			frame.ImageTransparency = FRAME_SELECTED_TRANSPARENCY
-		end)
-		frame.MouseLeave:connect(function()
-			frame.ImageTransparency = FRAME_DEFAULT_TRANSPARENCY
-		end)
 
+		--[[frame.MouseEnter:connect(function()
+				frame.ImageTransparency = FRAME_SELECTED_TRANSPARENCY
+			end)
+			frame.MouseLeave:connect(function()
+				frame.ImageTransparency = FRAME_DEFAULT_TRANSPARENCY
+			end)
+		end]]
```

Comment:
I have bad experiences with removing code and readding it. ;) I suppose it could just be removed since it isn't too important.